src/lib/WIFI/devWIFI.cpp

```diff
@@ -643,9 +652,13 @@ static void startWiFi(unsigned long now)
 
     // Set transmit power to minimum
     POWERMGNT::setPower(MinPower);
-    connectionState = wifiUpdate;
+
+    // Calling this means the LED and other devices will be notified of the change in state.
+    setWifiUpdateMode();
 
     DBGLN("Stopping Radio");
+    InBindingMode = false;
+    devicesTriggerEvent();
     Radio.End();
```

Comment:
Why is this in here twice? `setWifiUpdateMode` _and_ the body of `setWifiUpdateMode` copied here?

Reply:
Because I'm dumb!